tutorials/customizing-base-map-styles/index.md

```diff
@@ -0,0 +1,332 @@
+---
+title: Customizing base map styles
+description: This tutorial shows how to change the base map properties in Earth Engine, which is useful if you want a specific visualization for functionality/convenience.
+author: TC25
+tags: Earth Engine, Base Map, UI, Widgets
+date_published: 2019-09-16
+---
+
+[Open In Code Editor](https://code.earthengine.google.com/283c86dd3189788452b5fb64174a1b43)
+
+The user is going to learn how to change the options of the map object to create new base maps
+
+## The default maps in Earth Engine
+
+Earth Engine's base maps are those in Google's Map API. The default options include:
+
+- roadmap, which displays the default road map view,
+- satellite, which displays Google Earth satellite images,
+- hybrid, which displays a mixture of normal and satellite views, and
+- terrain, which displays a physical map based on terrain information.
+
+## Changing the basic map style
+
+We can start by changing the style of the base map. One easy fix is to invert the lightness to get a darker background, like so:
+
+```javascript
+
+var baseChange = [
+  "stylers": [ {"invert_lightness": true} ]
+]
+Map.setOptions("baseChange",{"baseChange":baseChange})
+
+```
+The main styler options include:
+
+ - hue: indicates the basic color
+ - lightness: indicates percentage change in brightness of an element
+ - saturation: indicates percentage change in basic color of an element
+ - gamma: Gamma correction of an element (0.01 and 10.0)
+ - invert_lightness: inverts the existing lightness
+ - visibility: Changes visibility options for an element (on, off, or simplified)
+ - color: Sets the color of an element (using RGB Hex Strings)
+ - weight: Sets the weight of a feature in pixels
```

Comment:
Nit: the initial letters should be consistently capitalized here (I suggest all lowercase).

Reply:
Done.